WORKSPACE

```diff
@@ -174,10 +174,16 @@ load(
 )
 
 haskell_register_ghc_nixpkgs(
-    attribute_path = "ghc",
+    attribute_path = "",
     compiler_flags = test_compiler_flags,
     haddock_flags = test_haddock_flags,
     locale_archive = "@glibc_locales//:locale-archive",
+
+    # The data-default-instances-old-locale package was is a reproduction sample for #1165
+    nix_file_content = """
+    with import <nixpkgs> {};
+    haskellPackages.ghcWithPackages(p: [p.data-default-instances-old-locale])
+    """,
```

Comment:
Would it work to pull in `data-default-instances-old-locale` through `stack_snapshot` instead and have a proper test case under `//tests` for this? That would also make the test case available in the bindist version and avoid increasing the difference between the nix and bindist versions.

Reply:
Nevermind, just realised, that `ls_modules` is not used on `stack_snapshot`...

Reply:
Well, for the "//tests" case, I have no idea, this is supposed to fail when parsing the package database, so I don't really know how we can write a "test" for it.
About `stack_snapshot`, I don't know either. Does `stack_snapshot` uses the package database?